cancel-old-builds.rst

```diff
@@ -0,0 +1,38 @@
+.. post:: October 4, 2022
+   :tags: builds, ci
+   :author: Manuel
+   :location: BCN
+
+.. meta::
+   :description lang=en:
+      A new feature that auto-cancels invalid builds was deployed. It improves the user experience,
+      reduces resource costs and also energy waste.
+
+Auto-cancel invalid builds
```

Comment:
This is a weird title that leaves me with multiple questions. Who is cancelling builds? What is an invalid build? etc. 

```suggestion
Platform update: Auto-canceling builds when pushing to the same branch twice
```

Reply:
👍🏼 --I removed the "Platform update" part because I think it's not needed and left the rest.